package.json

```diff
@@ -1708,16 +1708,16 @@
             }
         ],
         "viewsContainers": {
-            "activitybar": [
+            "panel": [
                 {
-                    "id": "jupyter",
+                    "id": "jupyter-variables",
```

Comment:
New ID here. We want this placed in the Panel by default, not in the activityBar. But for Don's new jupyter remote stuff (and other activity bar items) we do want them in the activity bar. So created a new ID and changed the placement to panel. Now it shows up by default where we want it.

Reply:
I'm confused. Is this going to be in the same place as the remote stuff or its own panel?

Reply:
Somewhat related, is this panel only going to show up for native notebooks? Or will it show up for old webviews too?

Reply:
@rchiodo Sorry if my comment didn't make this clear. The change here was that I wanted "Panel" positioning for my view, not "activityBar" positioning. Panel positioning puts it down next to the output and debug console windows. However I think that we want Don's remote stuff (and other general contributions) to live in the activityBar region. So I created a new viewContainer with a new ID here that explicitly docks in the Panel area. 

I see the "jupyter" viewContainer in the activityBar as our general tab / extensibility area, while this ID is specific to the variables. I'd assume that it's explicitly for just variables for now unless we run across more stuff that we want docked in the panel by default.

Reply:
And current plan was just for NativeNotebooks. I don't see a reason that we maybe couldn't change the old views to use this as well. But wasn't specced out in the current work. Not fully sure that I would want to for the old notebooks, since they work better with an explorer per document.

Reply:
Is there something that disables this view for the old stuff? (Hides it if you open an old webview or switch to the interactive window?)

Reply:
Hmm, currently it's just all hidden by the experiment switch. I was planning on using the existing editor context switches that we have for controlling visibility to show for native and hide if native is not active. Honestly though, with you and Joyce mentioning it. Maybe it's even easier to just have it always reflecting the active document, regardless of if native or not. Could give interactive / old webview customers a chance to use this in a panel if they prefer that.

Reply:
I think this needs to be experimented.
Personally I would prefer to have this displayed side by side, instead of displaying this at the bottom.
Ie. when user clicks on `show variables` button, I think we can display this view (so that it will be visible to the user).

@claudiaregio /cc
I think we should experiment on different placements.

Reply:
@DonJayamanne I'm not fully sure what you mean by side-by-side. Do you mean the activityBar? Right now I'm pretty sure that panel or activitybar are our only two default placement options. 

But on the plus side pretty easy to move this to a new location, and experimenting with that seems smart.